From b61011a08139ddfc7b6654e141373379aabfbe64 Mon Sep 17 00:00:00 2001
From: Richard Guy Briggs <rgb@redhat.com>
Date: Wed, 1 Oct 2014 21:35:56 -0400
Subject: [kernel] syscall.h: fix doc text for syscall_get_arch()

Message-id: <f99de1257ce8575b52f75de85a66fc4781de7af1.1412192166.git.rgb@redhat.com>
Patchwork-id: 96860
O-Subject: [RHEL7 PATCH V2 3/9] syscall.h: fix doc text for syscall_get_arch()
Bugzilla: 1053921
RH-Acked-by: Eric Paris <eparis@redhat.com>
RH-Acked-by: Paul Moore <pmoore@redhat.com>

syscall_get_arch() used to take a task as a argument.  It now uses
current.  Fix the doc text.

Upstream-commit: 1002d94
Signed-off-by: Richard Guy Briggs <rgb@redhat.com>
Signed-off-by: Eric Paris <eparis@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/asm-generic/syscall.h b/include/asm-generic/syscall.h
index d401e54..0c938a4 100644
--- a/include/asm-generic/syscall.h
+++ b/include/asm-generic/syscall.h
@@ -147,7 +147,7 @@ void syscall_set_arguments(struct task_struct *task, struct pt_regs *regs,
  *
  * Returns the AUDIT_ARCH_* based on the system call convention in use.
  *
- * It's only valid to call this when @task is stopped on entry to a system
+ * It's only valid to call this when current is stopped on entry to a system
  * call, due to %TIF_SYSCALL_TRACE, %TIF_SYSCALL_AUDIT, or %TIF_SECCOMP.
  *
  * Architectures which permit CONFIG_HAVE_ARCH_SECCOMP_FILTER must
-- 
1.7.1